lib/routes/liquipedia/cs-matches.ts

```diff
@@ -21,26 +22,52 @@ async function handler(ctx) {
     const rootUrl = 'https://liquipedia.net';
     const currentUrl = `${rootUrl}/counterstrike/${team}/Matches`;
 
-    const response = await got({
-        method: 'get',
-        url: currentUrl,
+    const response = await ofetch(currentUrl, {
+        headers: {
+            'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36',
+            Accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8',
+            'Accept-Language': 'en-US,en;q=0.5',
+            'Accept-Encoding': 'gzip, deflate',
+        },
```

Comment:
Why use this UA of a fixed version of Chrome on Windows? Does the site break on RSSHub's [default UA](https://github.com/DIYgod/RSSHub/blob/45ee587316dfc9939762331578083968fa05b1e6/lib/config.ts#L475) which is a randomised version of Chrome on macOS?